File: paging-sample-2/app/src/main/java/com/abubusoft/kripton/samples/paging/SampleApplication.java
package com.abubusoft.kripton.samples.paging;

import android.app.Application;

import com.abubusoft.kripton.android.KriptonLibrary;

public class SampleApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        KriptonLibrary.init(this);

        dataSource=BindCheeseDataSource.getInstance();
    }

    public BindCheeseDataSource getDataSource() {
        return dataSource;
    }

    private BindCheeseDataSource dataSource;
}

File: paging-with-network/app/src/main/java/com/android/example/paging/pagingwithnetwork/SampleApplication.kt
package com.android.example.paging.pagingwithnetwork

import android.app.Application
import com.abubusoft.kripton.android.KriptonLibrary

class SampleApplication: Application() {
    override fun onCreate() {
        super.onCreate()

        KriptonLibrary.init(this);
    }
}